noxfiles/dev_nox.py

```diff
@@ -30,6 +30,17 @@ def dev(session: Session) -> None:
         datastore for datastore in session.posargs if datastore in ALL_DATASTORES
     ] or None
 
+    if "child" in session.posargs:
```

Comment:
this can probably be streamlined a bit with overall nox workflow -- just wanted to get something quick in that allowed me to spin up the child env